registry/proxy/proxyblobstore.go

```diff
@@ -67,6 +69,29 @@ func (pbs *proxyBlobStore) copyContent(ctx context.Context, dgst digest.Digest,
 	return desc, nil
 }
 
+func (pbs *proxyBlobStore) doResponse(reqMethod string, ctx context.Context, dgst digest.Digest, writer io.Writer) error {
```

Comment:
`context.Context` should be first argument

This function name is also very generic for a function that does something very specific. Maybe `doServeBlobResponse` or something like that.